asset-transfer-basic/chaincode-javascript/.editorconfig

```diff
@@ -0,0 +1,16 @@
+#
```

Comment:
As this is specific to an editor, probably best not to include this.

Reply:
done.